src/gsp.py

```diff
@@ -58,12 +58,14 @@ async def get_all_available_forecasts(
     return forecasts
 
 
-@router.get("/forecast/{gsp_id}", response_model=Forecast)
+@router.get("/forecast/{gsp_id}", response_model=Union[Forecast, List[ForecastValue]])
 async def get_forecasts_for_a_specific_gsp(
     gsp_id: int,
     session: Session = Depends(get_session),
     historic: Optional[bool] = False,
-) -> Forecast:
+    only_forecast_values: Optional[bool] = False,
+    forecast_horizon_minutes: Optional[int] = None,
+) -> Union[Forecast, List[ForecastValue]]:
     """### Get one forecast for a specific GSP
 
     The return object is a solar forecast with GSP system details.
```

Comment:
Would you be able to update this documentation. I.e also mention about just get the Forecast Values. Hopefully its just copying stuff from the old documentation

Reply:
I just made a new issue to update the description. probably isn't necessary? I'll go ahead and update the description on this pull request.

Reply:
yea, lets keep it together, if thats ok